packages/db/src/controller/level.ts

```diff
@@ -175,6 +177,17 @@ export class LevelDbController implements IDatabaseController<Uint8Array, Uint8A
 
     this.metrics?.dbWriteItems.inc({bucket}, itemsRead);
   }
+
+  /** Add collect function to capture metric for db size */
+  private addDbSizeMetricCollector(metrics: ILevelDbControllerMetrics): void {
+    const minKey = Buffer.from([0x00]);
+    const maxKey = Buffer.from([0xff]);
+
+    metrics.dbSizeTotal.addCollect(async () => {
+      const dbSize = await this.approximateSize(minKey, maxKey);
```

Comment:
@wemeetagain calling `approximateSize` could throw an error if the database is closed for some reason. In that case the scraping the metrics from `/metrics` whould return a 500 error. Do you think we should just catch any errors here and ignore them?

Reply:
for now, will just catch errors and ignore them. The db package does not utitlize the logger at the moment but I think it should be fine for this use case to just ignore the error as I have not observed a single error as of now. If size a single call to `approximateSize` would fail it is also not an issue as it is called periodically anyways